src/main/xerus/monstercat/api/APIConnection.kt

```diff
@@ -200,11 +215,34 @@ class APIConnection(vararg path: String) : HTTPQuery<APIConnection>() {
 			return ConnectResult(connectsid, validity, session)
 		}
 		
+		fun login(username: String, password: String): Boolean {
+			val connection = APIConnection("v2", "signin")
+			val context = HttpClientContext()
+			connection.post(HttpPost(connection.uri).apply {
+				setHeader("Accept", "application/json")
+				setHeader("Content-type", "application/json")
+				entity = StringEntity("""{"email":"$username","password":"$password"}""")
+			}, context)
+
+			val code = connection.response?.statusLine?.statusCode ?: 0
+			logger.debug("code returned is $code")
```

Comment:
Either make a proper debug message or remove it

Reply:
Made better AND trace with 94d29ef1915da2c73fe8451d994bf5e47c56b101